SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
 #include "EngineClasses/SpatialLoadBalanceEnforcer.h"
+
```

Comment:
There's no need for the enforcer once we have USLB right?

Reply:
You still need it.

Reply:
Broadly it's our slightly strange use of partitions that necessitate it I think + possibly some lifecycle issues.
I've found it's irritatingly complicated to the point where it's hard to say what you do to be able to delete it.